src/adapters/Cornerstone/Segmentation_4X.js

```diff
@@ -629,6 +629,43 @@ function checkSEGsOverlapping(
     var groupsLen = PerFrameFunctionalGroupsSequence.length;
     var numberOfSegs = multiframe.SegmentSequence.length;
     var numberOfFrames = imageIds.length;
+    // NOTE:
+    // Assumption1 : numberOfSegs * numberOfFrames = groupsLen,
+    // i.e. for each frame we have a N PerFrameFunctionalGroupsSequence, where N is the numberOfSegs.
+    // Assumption2 : the order of the group sequence is = numberOfFrames of segmentation 1 +  numberOfFrames of segmentation 2 + ... + numberOfFrames of segmentation numberOfSegs
```

Comment:
Lets move this to jsDocs

Reply:
you are right, much cleaner and produce docs